### PR TITLE
homepage + overview pages

### PR DESCRIPTION
**Pull request by kidSoba**
This PR adds the following features / sites:

- fixes a bug, where the tutorial page would allow to be displayed, despite no being logged in
- homepage and sets it as page to be directed to when clicking "home" in the navbar
- overview-pages for the cards on the homepages
- navbar-highlight for the active page

### CODE REVIEW

File: templates/pages/QA.html [hunk 1]
```diff
@@ -217,7 +217,7 @@
                             <a href="https://www.ease-crc.org">
                                 <img id="ease-funding-logo"
                                 class=""
-                                src="{{ url_for('static', filename='img/ease.png') }}"
+                                src="{{ url_for('static', filename='img/Ease-Ease.svg') }}"
```
Comment (daniel86) [inline templates/pages/QA.html]: the naming of the file is odd.
the CRC is called EASE with all uppercase, so the leading uppercase character looks wrong.
Please either use all uppercase or all lowercase.
Also why is it "Ease-Ease" and not just "Ease"?

File: templates/pages/QA.html [hunk 2]
```diff
@@ -217,7 +217,7 @@
...
                                 alt="EASE Collaborative Research Group Logo">
```
Comment (daniel86) [inline templates/pages/QA.html]: "Collaborative Research Center" (CRC)

File: templates/pages/QA.html (hunk 1 above)
Comment (kidSoba) [reply templates/pages/QA.html]: Changed the name and committed it. The file is now called 'ease-crc.svg'

**Comment by kidSoba** [comment]
Since this hasn't been merged yet and it was requested to replace the neem-overview pages with some sort of system that would read out neem-git markdowns and render a page, I will close this pull-request and issue a seperate one once the feature is ready

**Pull request closed without merging**